core/tests/webdriverio_utils/waitFor.js

```diff
@@ -78,7 +78,7 @@ var invisibilityOf = async function(element, errorMessage) {
  * Consider adding this method after each browser.url() call.
  */
 var pageToFullyLoad = async function() {
-  var loadingMessage = $('.e2e-test-loading-fullpage');
+  var loadingMessage = await $('.e2e-test-loading-fullpage');
```

Comment:
I know we have discussed this. But can please also comment the reasoning behind it too here too so that it can be kept as a ref.

Reply:
Single selectors returns promise and we need to use `await` prefix to wait for it to resolve.

Reply:
I think you should also say about then why it's not being used in single selectors at other places.

Reply:
It works everywhere else because but here if we will not use await then it will throw `Unhandled promise rejection` error as we will directly pass promise to `invisibilityOf` but we need to resolve it before so added `await`.